test/Compiler.Dynamic.Tests/DynamicCompilationTests.cs

```diff
@@ -123,6 +125,7 @@ public async Task CompiledPageRuns(string test, params string[] pages)
                         .AddScriptManager()
                         .AddDynamicPages();
                     services.AddSingleton<IDataProtectionProvider, NoopDataProtector>();
+                    services.AddSingleton<IConfigureOptions<SiteMapOptions>, ConfigureSiteMapOptions>();
```

Comment:
is there a reason to register a configure options rather than just a callback?

Reply:
wanted to give a default option, but got rid of and cleaned up...